src/client.ts

```diff
@@ -95,8 +95,8 @@ export class Client {
   }
   /**
    * Sets the last transaction time of this client.
-   * @param time - the last transaction time to set.
-   * @throws Error if lastTxnTs is before the current lastTxn of the driver
+   * @param ts - the last transaction timestamp to set. If `ts` is less than the
+   *   existing `#lastTxnTs` value, then no change is made.
```

Comment:
```suggestion
   * @param ts - the last transaction timestamp to set, as microseconds since the epoch. If `ts` is less than the
   *   existing `#lastTxnTs` value, then no change is made.
```

Reply:
take it! one moment...